packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -673,6 +673,21 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
     return AtChopsImpl(atChopsKeys);
   }
 
+  @override
+  Future<Map<String, dynamic>> fetchEnrollmentRequests(
+      {String? appName, String? deviceName}) async {
+    EnrollVerbBuilder enrollBuilder = EnrollVerbBuilder()
+      ..operation = EnrollOperationEnum.list
+      ..appName = appName
+      ..deviceName = deviceName;
+
+    var response = await getRemoteSecondary()
+        ?.executeCommand(enrollBuilder.buildCommand(), auth: true);
```

Comment:
use executeAndParse instead of executeCommand

Reply:
executeAndParse() takes verbBuilder as input and enrollVerbBuilder as of now is not handled in AtLookUpImpl.executeVerb() [which is the method used by AtClient.executeVerb() and AtClient.executeAndParse()]. Because of this AtClient.executeVerb() and AtClient.executeAndParse() throw a request timed out when an EnrollVerbBuilder is passed as an input to AtClient.executeAndParse().